mephisto/providers/mock/wrap_crowd_source.js

```diff
@@ -53,3 +54,22 @@ function handleSubmitToProvider(task_data) {
     alert("The task has been submitted! Data: " + JSON.stringify(task_data))
     return true;
 }
+
+// Adding event listener instead of using window.onerror prevents the error to be caught twice
+window.addEventListener('error', function (event) {
+
+    if (event.error.hasBeenCaught !== undefined){
+    return false
+  }
+  event.error.hasBeenCaught = true
+  if (!auto_submit) {
+      if (confirm("Do you want to report the error?")) {
+          prompt('send the following error to the email address: '+
+          '[email address]', JSON.stringify(event.error.message))
```

Comment:
Sorry a bit out of context here - why are we asking for an email address?

Reply:
Pratik told me there should be a choice whether for the user to auto submit the error (wiring up `/log_error`) or show them an email and a copy-enabled version of the error message to send that error by themselves to an email address.

Reply:
Understood - generally we may have a case where we want the submit to be optional but not explicitly state the contact info (for a case where people may not want to expose their actual email, but instead get contacted through the site). So really there should be three options:
1. Auto-submit. Display the message "An error has occurred. This has been logged to the requester automatically."
2. Don't auto-submit. Display the message. "We think an error has occurred. If the task is broken, please send this context to the requester.
3. Don't auto-submit. Same message as the above, but with an email instead of "to the requester"